internal/mount/mount.go

```diff
@@ -57,26 +57,26 @@ func (m *Manager) Update(config models.DeviceConfigurationMessage) error {
 	}
 
 	// holds the directories mounted in the current _Update_ call.
-	// It helps avoiding mounting the same directory twice.
+	// It helps to avoid mounting the same directory twice.
 	alreadyMounted := make(map[string]struct{})
-	var merr *multierror.Error
+	var merr error
 	for _, mm := range config.Configuration.Mounts {
 		if _, found := alreadyMounted[mm.Directory]; found {
-			errf := fmt.Errorf("Directory '%s' has already been mounted. Skipping..", mm.Directory)
+			errf := fmt.Errorf("directory '%s' has already been mounted. Skipping directory", mm.Directory)
```

Comment:
Since we can now mount devices in /dev, should it still be called directory? Perhaps mount path?